src/components/CallToActionSection/CallToActionSection.tsx

```diff
@@ -14,23 +14,23 @@ const CallToActionSection = ({ className }: CalltoActionSectionProps) => {
   return (
     <section
       className={cn(
-        'relative pb-[100px] pt-[72px] text-primary xl:max-w-[81rem] xl:pb-0 xl:pt-20',
+        'relative pb-[6.25rem] pt-[4.5rem] text-primary xl:max-w-[81rem] xl:pb-0 xl:pt-20',
         className
       )}
     >
       <div className="absolute left-0 top-0">
-        <H1 className="w-fit bg-surface pl-[13px] pr-4 xl:pb-4">
+        <H1 className="w-fit bg-surface pl-[0.8125rem] pr-4 xl:pb-4">
           找不到
           <br className="hidden xl:block" />
           心目中的
           <br className="xl:hidden" />
           活動嗎？
         </H1>
-        <h1 className="bg-surface pb-4 pl-[13px] pr-4 text-5xl font-bold -tracking-[0.012em] xl:-mt-3 xl:w-fit">
+        <h1 className="bg-surface pb-4 pl-[0.8125rem] pr-4 text-5xl font-bold -tracking-[0.012em] xl:-mt-3 xl:w-fit">
           讓你來揪咖！
         </h1>
         <div className="w-fit bg-surface px-4 py-8">
-          <Separator className="h-[2px] w-12 bg-primary" />
+          <Separator className="h-0.5 w-12 bg-primary" />
```

Comment:
I've changed h-[2px] to h-0.5, and I apologize for pressing the wrong button on commit.

Reply:
It's okay, thank you🙏